baybe/utils/basic.py

```diff
@@ -54,6 +54,42 @@ def get_subclasses(cls: _C, recursive: bool = True, abstract: bool = False) -> l
     return subclasses
 
 
+def get_parent_classes(
```

Comment:
I don't mind too much, but for consistency, the function should rather be called `get_base_classes`, because it's counterpart is also *not* called `get_child_classes` and we speak of "base" classes everywhere else. @Scienfitz: opinions? If changed, the docstring should be adjusted accordingly

Reply:
agree (it was also named `base` in last meeting`

Reply:
Changed.